start.bat

```diff
@@ -0,0 +1,3 @@
+start cmd.exe /k "npm run start -w=ui"
+start cmd.exe /k "npm run start -w=server"
```

Comment:
run `main` and `metadata` nodes in separate consoles, because in one console it is uncomfortable to read it.
There are already `npm run main` and `npm run metadata`